### DISCUSSION TITLE
Welcome to sabbath-school-android Discussions!

### DISCUSSION THREAD

**Discussion by TinasheMzondiwa** [Adventech/sabbath-school-android#126] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anedroid** [comment]
Hey,

I am a member of the free software community. Me and my family often use this app. I have a request for you: can you publish it on F-Droid? F-Droid is a largest repository of free and open source software for Android. Every single app is automatically build from source to guarantee that apk doesn't have malicious functions added. It's possible to create your own F-Droid server too, but would be better if people could find Sabbath School app in main repository. F-Droid version should have removed all trackers including Google's and Facebook's.

Free software always gives to user 4 fundamental freedoms:
0. Freedom to run software as you wish
1. Freedom to study source code and modify software to do software do what you want. The necessary condition is access to the source code.
2. Freedom to redistribute copies as you wish
3. Freedom to share modified versions to help all community. Again, the necessary condition is access to the source code.

The GNU General Public Licence (GNU GPL) is used for most free software. GNU GPL makes use of the copyleft, that's reversed form of copyright. It allow user to this 4 freedoms, but doesn't allow to program become part of non-free software. All modified versions must be under GNU GPL licence too.

You don't have to remove app from Google Play store, it can be under double licence, GNU GPL and MIT. Publishing Sabbath School at F-Droid you can help some people to regain their freedom. And I hope you believe in freedom.

Please, consider my request.

**Reply by thebiblelover7** [reply]
I hope they will consider this. Right now, I have to manually update, since my phone is degoogled. Having it on F-droid would really help.

**Reply by boltsand** [reply]
I came here to ask about F-Droid support too.

**Reply by VitalikL** [reply]
Thanks @anedroid . Sorry for the late reply. I believe its definitely possible and since we only few external dependencies I will look into that in this calendar year.

**Comment by eowino-fintech** [comment]
Hi, for some reason the lesson quarterly for the 4th quarter hasn't been downloaded to the app yet. Is everything okay with it?
